api/apps/core/views.py

```diff
@@ -32,6 +33,7 @@ def get_queryset(self):
 class RequestViewSet(ObjectPermissionMixin, viewsets.ModelViewSet):
     permission_classes = [IsAuthenticated]
     serializer_class = RequestSerializer
+    queryset = Request.objects.all()
 
     def get_serializer_context(self):
         self.request.data["patient"] = self.request.user.pk
```

Comment:
musialem to przeoczyć wcześniej ale get_serializer_context używa się z deka inaczej:
[How serializer context works - StackOverflow](https://stackoverflow.com/questions/37275270/django-rest-framework-how-serializer-context-works)


```python
def get_serializer_context(self):
        return {
            'pk': self.request.user.pk
        }
```

Reply:
tylko, że to nie działa, a tamto tak, sprawdź u siebie